### ISSUE TITLE
Cloning fails becuse delegate refs is not found

### ISSUE THREAD

**Issue by geigerzaehler** [radicle-dev/radicle-cli#111]
Since #105 I can’t clone projects anymore for which I don’t have the delegate identity already. For example, the following fails for a fresh profile:

```plain
$ rad clone rad://willow.radicle.garden/hnrkjrrz963azhn3cfx4emb941zz5x6cf785y
Git version 2.36.1

Syncing 🌱 project rad:git:hnrkjrrz963azhn3cfx4emb941zz5x6cf785y from https://willow.radicle.garden/

!! Verifying project identity...

== Cloning failed error resolving latest head
== reference 'refs/namespaces/hnrkqrzmhso7a4hryucohqg4ge8wueshs77oy/refs/rad/id' not found; class=Reference (4); code=NotFound (-3)
```

The issue seems to by that in order to verify the project we need to fetch the identity of its delegate first.

**Comment by erak** [comment]
Thanks for reporting this! Can reproduce as well.

**Comment by cloudhead** [comment]
Yeah, a fix for this is coming!

**Comment by cloudhead** [comment]
Should be fixed by c17c41857f8f6772dbdad7fa5b589e595c87137d

**Comment by erak** [comment]
> Should be fixed by [c17c418](https://github.com/radicle-dev/radicle-cli/commit/c17c41857f8f6772dbdad7fa5b589e595c87137d)

Yes, it's fixed, so we can close this. @geigerzaehler Can you confirm?
